en/config/actuators.md

```diff
@@ -313,6 +313,51 @@ Configuring the Camera Capture / Trigger input requires a reboot to take effect
 You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
 These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
+### Output Functions
+
+Output functions are used to map the "logical functions" of an airframe, such as `Motor 1` or `Landing gear`, to physical outputs like FMU output pin 2.
+This makes it easy to use a particular output pin for almost any purpose.
+
+Some functions are only relevant to particular frames or output types, and will not be offered on others.
+
+Functions include: 
+
+- `Disabled`: Output has no assigned function.
+- `Constant_Min`: Output set to constant minimum value.
+- `Constant_Max`: Output is set to constant maximum value.
+- `Motor 1` to `Motor 12`: Output is indicated motor.
+  Only motors allowed for airframe are displayed.
+- `Servo 1` to `Servo 8`: Servo output.
+   These are further assigned a specific meaning based on airframe, such as "tilt servo", "left aileron".
+- `Offboard Acutator Set 1` to `Offboard Acutator Set 6`: MAVLink Payload output.
+- `Landing Gear`: Output is landing gear.
+- `Parachute`: Output is parachute.
```

Comment:
What else should we say here. I haven't checked other PR on Parachute yet, but we need to understand what value is set for this normally and what is used for the "trigger" value.

Reply:
It currently stays as Minimum (-1.0f) value, until a Failsafe is triggered, then the parachute gets deployed by setting the Maximum (1.0f) value.

So:
1. Move the slider to minimum and verify that parachute is in idle / stored state
2. Move the slider to maximum and verify that parachute is in deploy / open state
3. The parachute will get triggered automatically when the vehicle is in Failsafe mode

https://github.com/PX4/PX4-Autopilot/blob/55563eba49f87624d7b73356ea3d421fee7ec081/src/lib/mixer_module/functions.hpp#L244-L256

Reply:
Thanks. For this PR I will just update the minimum and maximum values.